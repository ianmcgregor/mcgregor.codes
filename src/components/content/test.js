import React from 'react';
import {render} from 'enzyme';
import Content from './';

const props = {
    title: 'Lorem ipsum',
    text: {
        childMarkdownRemark: {
            html: '<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio. Quisque volutpat mattis eros. Nullam malesuada erat ut turpis. Suspendisse urna nibh, viverra non, semper suscipit, posuere a, pede.</p>'
        }
    }
};

describe('Content component', () => {
    it('should render', () => {
        const el = render((
            <Content {...props}/>
        ));
        expect(el).toMatchSnapshot();
    });
});
