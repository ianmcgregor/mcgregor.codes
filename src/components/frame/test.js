import React from 'react';
import {render} from 'enzyme';
import Frame from './';

const props = {
    src: 'https://www.example.com'
};

describe('Frame component', () => {
    it('should render', () => {
        const el = render((
            <Frame
                {...props}
            />
        ));
        expect(el).toMatchSnapshot();
    });
});
