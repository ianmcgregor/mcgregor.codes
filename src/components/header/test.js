import React from 'react';
import {render} from 'enzyme';
import {MemoryRouter} from 'react-router-dom';
import Header from './';

const props = {
    links: [
        {
            label: 'Work',
            url: '/'
        },
        {
            label: 'About',
            url: '/about/'
        }
    ]
};

describe('Header component', () => {
    it('should render', () => {
        const el = render((
            <MemoryRouter>
                <Header {...props}/>
            </MemoryRouter>
        ));
        expect(el).toMatchSnapshot();
    });
});
