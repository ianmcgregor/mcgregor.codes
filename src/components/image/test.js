import React from 'react';
import {render} from 'enzyme';
import Image from './';

const props = {
    image: {
        id: 'c3Bh6RazkvSucssagW2GquS',
        title: 'comic-adventure',
        description: '',
        resolutions: {
            aspectRatio: 1.7777777777777777,
            width: 400,
            height: 225,
            src: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&q=50',
            srcSet: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&h=225&q=50 1x,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=600&h=338&q=50 1.5x,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=800&h=450&q=50 2x,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=1200&h=675&q=50 3x',
            srcWebp: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&q=50&fm=webp',
            srcSetWebp: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&h=225&q=50&fm=webp 1x,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=600&h=338&q=50&fm=webp 1.5x,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=800&h=450&q=50&fm=webp 2x,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=1200&h=675&q=50&fm=webp 3x'
        },
        file: {
            url: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png',
            fileName: 'comic-adventure.png',
            contentType: 'image/png'
        },
        sizes: {
            aspectRatio: 1.7777777777777777,
            src: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=800&q=100',
            srcSet: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=200&h=113&q=100 200w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&h=225&q=100 400w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=800&h=450&q=100 800w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=1200&h=675&q=100 1200w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=1600&h=900&q=100 1600w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=2400&h=1350&q=100 2400w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=2560&h=1440&q=100 2560w',
            srcWebp: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=800&q=100&fm=webp',
            srcSetWebp: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=200&h=113&q=100&fm=webp 200w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&h=225&q=100&fm=webp 400w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=800&h=450&q=100&fm=webp 800w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=1200&h=675&q=100&fm=webp 1200w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=1600&h=900&q=100&fm=webp 1600w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=2400&h=1350&q=100&fm=webp 2400w,\n//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=2560&h=1440&q=100&fm=webp 2560w',
            sizes: '(max-width: 800px) 100vw, 800px'
        },
        resize: {
            src: '//images.ctfassets.net/i8x1lkvuio7i/3Bh6RazkvSucssagW2GquS/a260280a7ec428b7a7ed1dec713df2a1/comic-adventure.png?w=400&fl=progressive&q=50',
            width: 400,
            height: 225,
            aspectRatio: 1.7777777777777777
        }
    }
};

describe('Image component', () => {
    it('should render', () => {
        const el = render((
            <Image {...props}/>
        ));
        expect(el).toMatchSnapshot();
    });
});
