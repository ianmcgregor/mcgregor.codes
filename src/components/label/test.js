import React from 'react';
import {render} from 'enzyme';
import Label from './';

const props = {
    text: 'Lorem ipsum'
};

describe('Label component', () => {
    it('should render', () => {
        const el = render((
            <Label {...props}/>
        ));
        expect(el).toMatchSnapshot();
    });
});
