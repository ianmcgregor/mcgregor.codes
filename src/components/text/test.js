import React from 'react';
import {render} from 'enzyme';
import Text from './';

const props = {
    text: {
        childMarkdownRemark: {
            html: '<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio. Quisque volutpat mattis eros. Nullam malesuada erat ut turpis. Suspendisse urna nibh, viverra non, semper suscipit, posuere a, pede.</p>'
        }
    }
};

describe('Text component', () => {
    it('should render', () => {
        const el = render((
            <Text {...props}/>
        ));
        expect(el).toMatchSnapshot();
    });
});
