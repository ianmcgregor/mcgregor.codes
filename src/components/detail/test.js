import React from 'react';
import {render} from 'enzyme';
import {MemoryRouter} from 'react-router-dom';
import {LinkStub} from '../../../test/helpers';
import Detail from './';

const props = {
    type: 'ContentfulProject',
    __typename: 'ContentfulProject',
    id: 'c2OLhcgLiesoUqs48yMa0co',
    title: 'Google â€” Inside Abbey Road',
    year: 2015,
    image: {
        id: 'c3JDzQqutDa6uE6oOs22AEO',
        title: 'abbey road 10',
        description: '',
        resolutions: {
            aspectRatio: 1.7777777777777777,
            width: 400,
            height: 225,
            src: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&q=50',
            srcSet: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&h=225&q=50 1x,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=600&h=338&q=50 1.5x,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=800&h=450&q=50 2x,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1200&h=675&q=50 3x',
            srcWebp: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&q=50&fm=webp',
            srcSetWebp: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&h=225&q=50&fm=webp 1x,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=600&h=338&q=50&fm=webp 1.5x,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=800&h=450&q=50&fm=webp 2x,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1200&h=675&q=50&fm=webp 3x'
        },
        file: {
            url: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png',
            fileName: 'abbey_road_10.png',
            contentType: 'image/png'
        },
        sizes: {
            aspectRatio: 1.7777777777777777,
            src: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=800&q=100',
            srcSet: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=200&h=113&q=100 200w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&h=225&q=100 400w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=800&h=450&q=100 800w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1200&h=675&q=100 1200w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1600&h=900&q=100 1600w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1920&h=1080&q=100 1920w',
            srcWebp: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=800&q=100&fm=webp',
            srcSetWebp: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=200&h=113&q=100&fm=webp 200w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&h=225&q=100&fm=webp 400w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=800&h=450&q=100&fm=webp 800w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1200&h=675&q=100&fm=webp 1200w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1600&h=900&q=100&fm=webp 1600w,\n//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=1920&h=1080&q=100&fm=webp 1920w',
            sizes: '(max-width: 800px) 100vw, 800px'
        },
        resize: {
            src: '//images.ctfassets.net/i8x1lkvuio7i/3JDzQqutDa6uE6oOs22AEO/9f09cad5e7504b8aeed7697d5c8832fa/abbey_road_10.png?w=400&fl=progressive&q=50',
            width: 400,
            height: 225,
            aspectRatio: 1.7777777777777777
        }
    },
    video: {
        id: 'c4JelMRzw1WCsoMGyKQkuUO',
        title: 'abbey road',
        description: '',
        file: {
            url: '//videos.ctfassets.net/i8x1lkvuio7i/4JelMRzw1WCsoMGyKQkuUO/c5c42ec2fd7a6b6b273e2e35b6f32649/abbey_road.mp4',
            fileName: 'abbey_road.mp4',
            contentType: 'video/x-m4v'
        }
    },
    text: {
        childMarkdownRemark: {
            html: '<p>I enjoyed pushing the Web Audio API to its limits on this virtual tour of Abbey Road studios which also entailed lots of WebGL and intricate UI work. I led the development over its year-long gestation. It was well received and won the <a href="https://thefwa.com/cases/inside-abbey-road">FWA Site of the Month.</a> and the Webby Award in the music category.</p>'
        }
    },
    link: 'https://insideabbeyroad.withgoogle.com/',
    links: [
        {
            title: 'Comic Adventure',
            slug: 'comic_adventure'
        },
        {
            title: 'Inside Abbey Road',
            slug: 'inside_abbey_road'
        },
        {
            title: 'Manchester City',
            slug: 'manchester_city'
        },
        {
            title: 'Den Automation',
            slug: 'den_automation'
        },
        {
            title: 'Pokemon Go Live',
            slug: 'pokemon_go_live'
        },
        {
            title: 'Kindernauts',
            slug: 'kindernauts'
        },
        {
            title: 'My Time Is Now',
            slug: 'my_time_is_now'
        },
        {
            title: 'Revolutions in Sound',
            slug: 'revolutions_in_sound'
        },
        {
            title: 'Primo Toys',
            slug: 'primo_toys'
        },
        {
            title: 'Chance Chanel',
            slug: 'chance_chanel'
        },
        {
            title: 'Monty\'s Christmas',
            slug: 'monty_s_christmas'
        },
        {
            title: 'Made',
            slug: 'made'
        },
        {
            title: 'W+K London',
            slug: 'w_k_london'
        },
        {
            title: 'Carousel',
            slug: 'carousel'
        },
        {
            title: 'Unite All Originals',
            slug: 'unite_all_originals'
        },
        {
            title: 'Urban Tour',
            slug: 'urban_tour'
        },
        {
            title: 'Kino',
            slug: 'kino'
        },
        {
            title: 'Happy Inside',
            slug: 'happy_inside'
        },
        {
            title: 'One Life',
            slug: 'one_life'
        },
        {
            title: 'Diesel Island',
            slug: 'diesel_island'
        },
        {
            title: 'Dark Ride',
            slug: 'dark_ride'
        },
        {
            title: 'A Hundred Lovers',
            slug: 'a_hundred_lovers'
        }
    ],
    slug: 'comic_adventure'
};

describe('Detail component', () => {
    it('should render', () => {
        const el = render((
            <MemoryRouter>
                <Detail
                    {...props}
                    LinkComponent={LinkStub}
                />
            </MemoryRouter>
        ));
        expect(el).toMatchSnapshot();
    });
});
