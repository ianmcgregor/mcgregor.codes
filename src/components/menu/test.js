import React from 'react';
import {render} from 'enzyme';
import {MemoryRouter} from 'react-router-dom';
import {LinkStub} from '../../../test/helpers';
import Menu from './';

const props = {
    links: [
        {
            label: 'Work',
            url: '/'
        },
        {
            label: 'About',
            url: '/about/'
        }
    ]
};

describe('Menu component', () => {
    it('should render', () => {
        const menu = render((
            <MemoryRouter>
                <Menu
                    {...props}
                    LinkComponent={LinkStub}
                />
            </MemoryRouter>
        ));
        expect(menu).toMatchSnapshot();
    });
});
