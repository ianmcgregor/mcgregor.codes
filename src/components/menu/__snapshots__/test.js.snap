// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should render 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "sc-ifAKCX hbKAms",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-EHOje cyvMVa",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bZQynM gAjREB",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "sc-gzVnrw fbHcEw",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "href": "#",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-bdVaJa fGHDiW",
                        },
                        "children": Array [
                          Object {
                            "attribs": Object {
                              "class": "sc-dnqmqq kGfHXs",
                            },
                            "children": Array [
                              Object {
                                "attribs": Object {
                                  "class": "sc-htpNat csPHbj",
                                },
                                "children": Array [
                                  Object {
                                    "attribs": Object {
                                      "class": "sc-bxivhb jZLiij",
                                    },
                                    "children": Array [
                                      Object {
                                        "data": "Work",
                                        "next": null,
                                        "parent": [Circular],
                                        "prev": null,
                                        "type": "text",
                                      },
                                    ],
                                    "name": "span",
                                    "namespace": "http://www.w3.org/1999/xhtml",
                                    "next": null,
                                    "parent": [Circular],
                                    "prev": null,
                                    "type": "tag",
                                    "x-attribsNamespace": Object {
                                      "class": undefined,
                                    },
                                    "x-attribsPrefix": Object {
                                      "class": undefined,
                                    },
                                  },
                                ],
                                "name": "span",
                                "namespace": "http://www.w3.org/1999/xhtml",
                                "next": null,
                                "parent": [Circular],
                                "prev": null,
                                "type": "tag",
                                "x-attribsNamespace": Object {
                                  "class": undefined,
                                },
                                "x-attribsPrefix": Object {
                                  "class": undefined,
                                },
                              },
                            ],
                            "name": "span",
                            "namespace": "http://www.w3.org/1999/xhtml",
                            "next": null,
                            "parent": [Circular],
                            "prev": null,
                            "type": "tag",
                            "x-attribsNamespace": Object {
                              "class": undefined,
                            },
                            "x-attribsPrefix": Object {
                              "class": undefined,
                            },
                          },
                        ],
                        "name": "div",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                        },
                      },
                    ],
                    "name": "a",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "href": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "href": undefined,
                    },
                  },
                ],
                "name": "li",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "class": "sc-gzVnrw fbHcEw",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "href": "#",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-bdVaJa fGHDiW",
                          },
                          "children": Array [
                            Object {
                              "attribs": Object {
                                "class": "sc-iwsKbI yIKfu",
                              },
                              "children": Array [
                                Object {
                                  "attribs": Object {
                                    "class": "sc-htpNat csPHbj",
                                  },
                                  "children": Array [
                                    Object {
                                      "attribs": Object {
                                        "class": "sc-bxivhb jZLiij",
                                      },
                                      "children": Array [
                                        Object {
                                          "data": "About",
                                          "next": null,
                                          "parent": [Circular],
                                          "prev": null,
                                          "type": "text",
                                        },
                                      ],
                                      "name": "span",
                                      "namespace": "http://www.w3.org/1999/xhtml",
                                      "next": null,
                                      "parent": [Circular],
                                      "prev": null,
                                      "type": "tag",
                                      "x-attribsNamespace": Object {
                                        "class": undefined,
                                      },
                                      "x-attribsPrefix": Object {
                                        "class": undefined,
                                      },
                                    },
                                  ],
                                  "name": "span",
                                  "namespace": "http://www.w3.org/1999/xhtml",
                                  "next": null,
                                  "parent": [Circular],
                                  "prev": null,
                                  "type": "tag",
                                  "x-attribsNamespace": Object {
                                    "class": undefined,
                                  },
                                  "x-attribsPrefix": Object {
                                    "class": undefined,
                                  },
                                },
                              ],
                              "name": "span",
                              "namespace": "http://www.w3.org/1999/xhtml",
                              "next": null,
                              "parent": [Circular],
                              "prev": null,
                              "type": "tag",
                              "x-attribsNamespace": Object {
                                "class": undefined,
                              },
                              "x-attribsPrefix": Object {
                                "class": undefined,
                              },
                            },
                          ],
                          "name": "div",
                          "namespace": "http://www.w3.org/1999/xhtml",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                          "x-attribsNamespace": Object {
                            "class": undefined,
                          },
                          "x-attribsPrefix": Object {
                            "class": undefined,
                          },
                        },
                      ],
                      "name": "a",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "href": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "href": undefined,
                      },
                    },
                  ],
                  "name": "li",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "class": "sc-gzVnrw fbHcEw",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "href": "#",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-bdVaJa fGHDiW",
                        },
                        "children": Array [
                          Object {
                            "attribs": Object {
                              "class": "sc-iwsKbI yIKfu",
                            },
                            "children": Array [
                              Object {
                                "attribs": Object {
                                  "class": "sc-htpNat csPHbj",
                                },
                                "children": Array [
                                  Object {
                                    "attribs": Object {
                                      "class": "sc-bxivhb jZLiij",
                                    },
                                    "children": Array [
                                      Object {
                                        "data": "About",
                                        "next": null,
                                        "parent": [Circular],
                                        "prev": null,
                                        "type": "text",
                                      },
                                    ],
                                    "name": "span",
                                    "namespace": "http://www.w3.org/1999/xhtml",
                                    "next": null,
                                    "parent": [Circular],
                                    "prev": null,
                                    "type": "tag",
                                    "x-attribsNamespace": Object {
                                      "class": undefined,
                                    },
                                    "x-attribsPrefix": Object {
                                      "class": undefined,
                                    },
                                  },
                                ],
                                "name": "span",
                                "namespace": "http://www.w3.org/1999/xhtml",
                                "next": null,
                                "parent": [Circular],
                                "prev": null,
                                "type": "tag",
                                "x-attribsNamespace": Object {
                                  "class": undefined,
                                },
                                "x-attribsPrefix": Object {
                                  "class": undefined,
                                },
                              },
                            ],
                            "name": "span",
                            "namespace": "http://www.w3.org/1999/xhtml",
                            "next": null,
                            "parent": [Circular],
                            "prev": null,
                            "type": "tag",
                            "x-attribsNamespace": Object {
                              "class": undefined,
                            },
                            "x-attribsPrefix": Object {
                              "class": undefined,
                            },
                          },
                        ],
                        "name": "div",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                        },
                      },
                    ],
                    "name": "a",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "href": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "href": undefined,
                    },
                  },
                ],
                "name": "li",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "sc-gzVnrw fbHcEw",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "href": "#",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-bdVaJa fGHDiW",
                          },
                          "children": Array [
                            Object {
                              "attribs": Object {
                                "class": "sc-dnqmqq kGfHXs",
                              },
                              "children": Array [
                                Object {
                                  "attribs": Object {
                                    "class": "sc-htpNat csPHbj",
                                  },
                                  "children": Array [
                                    Object {
                                      "attribs": Object {
                                        "class": "sc-bxivhb jZLiij",
                                      },
                                      "children": Array [
                                        Object {
                                          "data": "Work",
                                          "next": null,
                                          "parent": [Circular],
                                          "prev": null,
                                          "type": "text",
                                        },
                                      ],
                                      "name": "span",
                                      "namespace": "http://www.w3.org/1999/xhtml",
                                      "next": null,
                                      "parent": [Circular],
                                      "prev": null,
                                      "type": "tag",
                                      "x-attribsNamespace": Object {
                                        "class": undefined,
                                      },
                                      "x-attribsPrefix": Object {
                                        "class": undefined,
                                      },
                                    },
                                  ],
                                  "name": "span",
                                  "namespace": "http://www.w3.org/1999/xhtml",
                                  "next": null,
                                  "parent": [Circular],
                                  "prev": null,
                                  "type": "tag",
                                  "x-attribsNamespace": Object {
                                    "class": undefined,
                                  },
                                  "x-attribsPrefix": Object {
                                    "class": undefined,
                                  },
                                },
                              ],
                              "name": "span",
                              "namespace": "http://www.w3.org/1999/xhtml",
                              "next": null,
                              "parent": [Circular],
                              "prev": null,
                              "type": "tag",
                              "x-attribsNamespace": Object {
                                "class": undefined,
                              },
                              "x-attribsPrefix": Object {
                                "class": undefined,
                              },
                            },
                          ],
                          "name": "div",
                          "namespace": "http://www.w3.org/1999/xhtml",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                          "x-attribsNamespace": Object {
                            "class": undefined,
                          },
                          "x-attribsPrefix": Object {
                            "class": undefined,
                          },
                        },
                      ],
                      "name": "a",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "href": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "href": undefined,
                      },
                    },
                  ],
                  "name": "li",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "ul",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "nav",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
